tests/Api/StatsTest.php

```diff
@@ -42,6 +42,22 @@ public function testTotal($queryParameters, $responseData)
         $this->assertInstanceOf(TotalResponse::class, $total);
         $this->assertCount(count($responseData['stats']), $total->getStats());
         $this->assertContainsOnlyInstancesOf(TotalResponseItem::class, $total->getStats());
+
+        $event = $queryParameters['event'];
+        $responseStat = $total->getStats()[0];
+        $statGetter = 'get'.ucwords($event);
+
+        if ('failed' !== $event) {
+            $expectedTotal = $responseData['stats'][0][$event]['total'];
+            $actualTotal = $responseStat->$statGetter()['total'];
+        }
+
+        if ('failed' == $event) {
```

Comment:
A trivial point, buy maybe `===` is better here, as we intend to get a match

```suggestion
        if ('failed' === $event) {
```

Reply:
Sure, not a problem, I will push that change up shortly. 
Thanks for reviewing!